ports/azure_sdk.md

```diff
@@ -0,0 +1,341 @@
+There are a few steps to follow:
+
+ 1. Create and set up `vibr0` bridge on a host using `qemu-bridge-helper`:
+
+  - Install the required package and ensure that `libvirtd` is running:
+
+  ```
+  sudo apt-get update
+  sudo apt-get install qemu-system-common
+  systemctl enable libvirtd.service
+  systemctl start libvirtd.service
+  ```
+
+  - Start the default network bridge, and configure it to run on startup.
+
+  ```
+  sudo virsh net-autostart --network default
+  sudo virsh net-start --network default
+  ```
+
+  - After that verify that the IP range `192.168.122.1/24` is reported by the `vibr0` bridge:
+
+  ```
+  ip addr show virbr0
+  ```
+
+  - The expected output:
+
+  ```
+   virbr0: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1500 qdisc noqueue state DOWN group default qlen 1000
+      link/ether xx:xx:xx:xx:xx:xx brd ff:ff:ff:ff:ff:ff
+      inet 192.168.122.1/24 brd 192.168.122.255 scope global virbr0
+         valid_lft forever preferred_lft forever
+  ```
+
+  - Set up `qemu-bridge-helper` (`chmod` is used here to allow running qemu without root privileges)
+
+  ```
+  echo "allow virbr0" > /etc/qemu/bridge.conf
+  sudo chmod a+rw /etc/qemu/bridge.conf
+  ```
+
+  - The `/etc/qemu` directory may not exist, then create it and provide the required privileges:
+
+  ```
+  sudo mkdir /etc/qemu
+  sudo chmod a+rw /etc/qemu
+  echo "allow virbr0" > /etc/qemu/bridge.conf
+  sudo chmod a+rw /etc/qemu/bridge.conf
+  ```
+
+  <img src="_images/azure_sdk_vibr_setup.png" width="700px">
+
+  - Sources: https://apiraino.github.io/qemu-bridge-networking/, https://mike42.me/blog/2019-08-how-to-use-the-qemu-bridge-helper-on-debian-10
+
+ 2. Launch `qemu` using a starting script with `net` suffix:
+
+  ```
+  ./scripts/ia32-generic-qemu-net.sh
+  ```
+
+ 3. Configure network and run `ash` (Busybox applet) using `rc` script:
+
+  - Note: `IP` is assigned using `DHCP`, there are also other possibilities, for more please check the configuration file located in `_projects/ia32-generic-qemu/rootfs-overlay/etc/rc.conf.d/network`
+
+  - Note: There are also executed other programs, for more information please check the content of the `rc` file for `ia32-generic-qemu` in `_projects/ia32-generic-qemu/rootfs-overlay/etc/rc`
+
+  ```
+  /linuxrc
+  ```
+
+  - As You can see, the advanced version of `Phoenix-RTOS` with `POSIX` shell has been started:
+
+  <img src="_images/azure_linuxrc.png" width="700px">
+
+  - Now You can check the internet connection using the `ping` applet.
+
+  - To make the configuration complete please set the current date using the `date` applet, like below:
+
+  <img src="_images/azure_date.png" width="700px">
+
+  - Note: The required argument entered after `@` is EPOCH (POSIX time format) - You can gt it for example from [here](https://www.epochconverter.com/).
+
+
+
+### Running the IoThub Client sample
+
+Providing the internet connection is working and the current date is set, You can run the built `iothub_ll_telemetry_sample`:
```

Comment:
```suggestion
Provided that internet connection is working and the current date is set, you can run the build `iothub_ll_telemetry_sample`:
```

Reply:
I meant sample, which have been built. However, removing it seems to be a better idea.